openedx/features/content_type_gating/field_override.py

```diff
@@ -46,6 +46,14 @@ def get(self, block, name, default):
 
         if original_group_access is None:
             original_group_access = {}
+
+            # For Feature Based Enrollments, we want to inherit group access configurations
+            # from parent blocks. The use case is to allow granting access
+            # to all graded problems in a unit at the unit level
+            merged_group_access = block.get_parent().merged_group_access
+            if merged_group_access and CONTENT_GATING_PARTITION_ID in merged_group_access:
+                return original_group_access
```

Comment:
I think you need to dedent this. Right now, it only fires if `original_group_access` is None.

Reply:
Never mind, this is correct.